lib/searchkick.rb

```diff
@@ -24,13 +24,11 @@ class UnsupportedVersionError < StandardError; end
   class InvalidQueryError < Elasticsearch::Transport::Transport::Errors::BadRequest; end
 
   class << self
-    attr_accessor :callbacks
```

Comment:
Not sure about getting rid of this - nothing in the gem appears to use it, but I don't know if people using the gem would be checking its value. You could argue that this should always have been a private implementation and the public API is to use `callbacks?`, and therefore anyone inspecting `callbacks` is _wrong_. You could also just `alias :callbacks :callbacks?`